test/test-except.c

```diff
@@ -56,6 +56,10 @@ main(void)
 
     __asm__(".word 0x00000000");
 
+#elif defined(__aarch64__)
```

Comment:
I think the following should be a portable option: 

```
#if __has_builtin(__builtin_trap)
__builtin_trap()
#endif
```

Reply:
good point. Although, right now, that whole ifdef chain is also signaling which platforms have the necessary underlying exception support.